packages/core/src/components/svgs/VerifiedTick.ts

```diff
@@ -9,7 +9,7 @@ export default function VerifiedTick(props: Record<string, unknown> = {}) {
 	return html`<svg
 		${spread(props)}
 		class="discord-application-tag-verified"
-		aria-label="Verified Bot"
+		aria-label="Verified App"
```

Comment:
It doesn't seem to have been renamed here, but I'm guessing it will eventually 😅 
![image](https://github.com/skyra-project/discord-components/assets/24852502/ee67e2c5-b38b-4f0a-a3c6-cfd63ef61fb2)

Reply:
Want me to change that to make it future proof or let it stay?